ci-operator/jobs/openstack-k8s-operators/cinder-operator/openstack-k8s-operators-cinder-operator-master-presubmits.yaml

```diff
@@ -1,7 +1,7 @@
 presubmits:
   openstack-k8s-operators/cinder-operator:
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
The `cinder-operator-build-deploy` job should have `always_run: false`, since it's duplicated by the kuttl one